config/_templates/dataset/yaak.yaml

```diff
@@ -20,9 +20,9 @@ sources:
     (@=source_id@):
       index_column: meta/ImageMetadata.(@=source_id@)/frame_idx
       source:
-        _target_: rbyte.io.FfmpegFrameSource
+        _target_: rbyte.io.TorchCodecFrameSource
         path: ${data_dir}/(@=input_id@)/(@=source_id@).pii.mp4
-        resize_shorter_side: 324
+        device: cpu
```

Comment:
Since no resize is happening, decoder outputs `1080 x 1920`

so https://github.com/yaak-ai/rbyte/blob/9f27baf78cf5688ec800e5958a9ece48b7fb3d90/config/_templates/logger/rerun/yaak.yaml#L36-L37

should be adjusted. Note that it slows rerun significantly

Also, once carGPT is bumped, resize transform should be added

https://github.com/yaak-ai/carGPT/blob/41b9e025b1f931df2ff3cfb6a0cc3778456ca0fd/config/model/control_transformer/raw.yaml#L31-L32

Reply:
Makes sense we wanted cargpt/requester to handle transforms anyway